docs/src/appendix/sncast/call.md

```diff
@@ -31,6 +31,12 @@ Optional.
 Inputs to the function, represented by a list of space-delimited values, e.g. `0x1 2 0x3`.
 Calldata arguments may be either 0x hex or decimal felts.
 
+## `--arguments`
```

Comment:
Let's link to a docs section explaining how arguments handling works. Here and in changes below.